lib/src/io/node.dart

```diff
@@ -80,6 +90,22 @@ _readFile(String path, [String encoding]) {
   }
 }
 
+Future<String> readStdin() async {
+  var completer = new Completer<String>();
+  var contents = new StringBuffer();
+  _stdin.on('data', allowInterop((buf) {
+    contents.write(UTF8.decode(buf));
```

Comment:
Done. Unfortunately I found no [notion of "system encoding" in Node](https://www.google.com/search?q=node+system+encoding). The closest thing to a default is that Buffer [defaults to 'utf8' all over the place](https://nodejs.org/api/buffer.html#buffer_buffers_and_character_encodings).